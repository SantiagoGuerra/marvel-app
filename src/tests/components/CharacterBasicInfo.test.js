
const React = require('react');
const { shallow } = require("enzyme");
const { default: CharacterBasicInfo } = require("../../components/CharacterBasicInfo");

describe('Test <CharacterBasicInfo /> Component', () => {
  test('should render the component correctly', () => {
    
    const wrapper = shallow(<CharacterBasicInfo />)


    expect( wrapper ).toMatchSnapshot()

    
  })
  
})
