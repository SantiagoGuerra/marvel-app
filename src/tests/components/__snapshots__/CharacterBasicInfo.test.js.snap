// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <CharacterBasicInfo /> Component should have data as parameter 1`] = `
<Fragment>
  <Box
    direction="column"
    gap="medium"
    wrap={true}
  />
  <CharacterDescription
    description=""
    name=""
    thumbnail={
      Object {
        "extension": "",
        "path": "",
      }
    }
  />
  <Tabs
    comics={Object {}}
    events={Object {}}
    series={Object {}}
    stories={Object {}}
  />
</Fragment>
`;

exports[`Test <CharacterBasicInfo /> Component should render the component correctly 1`] = `
<Fragment>
  <Box
    direction="column"
    gap="medium"
    wrap={true}
  />
  <CharacterDescription
    description=""
    name=""
    thumbnail={
      Object {
        "extension": "",
        "path": "",
      }
    }
  />
  <Tabs
    comics={Object {}}
    events={Object {}}
    series={Object {}}
    stories={Object {}}
  />
</Fragment>
`;
